Firestore/core/src/local/leveldb_key.h

```diff
@@ -127,6 +127,13 @@ namespace local {
 //   - user_id: string
 //   - largest_batch_id: model::BatchId
 //   - document_key: ResourcePath
+//
+// document_overlays_collection_index:
+//   - table_name: "document_overlays_collection_index"
+//   - user_id: string
+//   - collection: ResourcePath
+//   - largest_batch_id: model::BatchId
+//   - document_key: ResourcePath
```

Comment:
As stated above, maybe just use the document_id.

Reply:
Done. This comment was updated to replace `document_key: ResourcePath` with `document_id: string`.